resources/js/components/FormField.vue

```diff
@@ -73,7 +77,7 @@
 						return;
 					}
 
-					if(dependency.hasOwnProperty('value') && this.dependencyValues[dependency.field] !== dependency.value) {
+					if(dependency.hasOwnProperty('value') && this.dependencyValues[dependency.field] != dependency.value) {
```

Comment:
I'm not sure I would say this change would be good, since first you as a dev should know what the value is that you are checking against, but also there are some odd type behaviors in JS that I don't think we want to mess with. 
Is there a specific reason to convert this from `!==` to a `!=`?